lib/IDE/Refactoring.cpp

```diff
@@ -2924,7 +2924,7 @@ collectAvailableRefactoringsAtCursor(SourceFile *SF, unsigned Line,
   // Prepare the tool box.
   ASTContext &Ctx = SF->getASTContext();
   SourceManager &SM = Ctx.SourceMgr;
-  DiagnosticEngine DiagEngine(SM);
+  DiagnosticEngine DiagEngine(SM, "/path");
```

Comment:
I'm not sure how I'll be passing the default diagnostic messages path to `lib/IDE/Refactoring.cpp`

Reply:
/cc @rintaro

Reply:
apparently the methods in Refactoring.cpp are called from tools in `tools/` so I guess passing the `MainExecutablePath` from there will be the way to go. 
cc @rintaro, @nkcsgexi